crates/bevy_ui/src/layout/mod.rs

```diff
@@ -315,29 +316,22 @@ pub fn resolve_outlines_system(
 
 #[inline]
 /// Round `value` to the nearest whole integer, with ties (values with a fractional part equal to 0.5) rounded towards positive infinity.
-fn round_ties_up(value: f32) -> f32 {
```

Comment:
I'm most nervous about this change. Can you split this from the other change so we can discuss it separately and see what the relative performance impact is?

Reply:
the only difference from the previous function is that values in `[-0.5, 0[` used to round to negative 0, and will now round to positive 0

Reply:
the main concern lies in potential precision loss. If the value happens to be 0.49999997 or greater than 8388609, there could be a one-unit error compared to the previous function.I totally agree that we should split this into another PR because of the potential error although IMO its not essential for ui.